File: app/src/main/java/net/osdn/gokigen/a01d/camera/panasonic/wrapper/IPanasonicCameraHolder.java
package net.osdn.gokigen.a01d.camera.panasonic.wrapper;



import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import net.osdn.gokigen.a01d.camera.ICameraChangeListener;

public interface IPanasonicCameraHolder
{
    void detectedCamera(@NonNull IPanasonicCamera camera);
    void prepare();
    void startRecMode();
    void startEventWatch(@Nullable ICameraChangeListener listener);
}

File: app/src/main/java/net/osdn/gokigen/a01d/camera/panasonic/wrapper/IPanasonicCameraHolder.kt
package net.osdn.gokigen.a01d.camera.panasonic.wrapper

import net.osdn.gokigen.a01d.camera.ICameraChangeListener

interface IPanasonicCameraHolder
{
    fun detectedCamera(camera: IPanasonicCamera)
    fun prepare()
    fun startRecMode()
    fun startEventWatch(listener: ICameraChangeListener?)
}
